auth/jwt.go

```diff
@@ -4,7 +4,7 @@ import (
 	"time"
 	"sync"
 	"net/http"
-	"github.com/dgrijalva/jwt-go"
+	jwt2 "github.com/dgrijalva/jwt-go"
```

Comment:
Why `jwt2`?

Reply:
at first it looked like there was a conflict of the jwt name, but you're right, it's not necessary :thinking: